Dockerfile-refresher

```diff
@@ -38,6 +38,8 @@ WORKDIR /
 COPY --from=builder /etc/nsswitch.conf.build /etc/nsswitch.conf
 COPY --from=builder /usr/share/zoneinfo /usr/share/zoneinfo
 COPY --from=builder /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/
+RUN mkdir /licenses
```

Comment:
This should be redundant as `COPY` creates the directory if it doesn't exist.

Reply:
good point, done and tested, thanks!